### PR TITLE
Coredev docs update

### PR DESCRIPTION
**Pull request by sunew**

### CODE REVIEW

File: docs/git.rst [hunk 1]
```diff
@@ -560,6 +560,15 @@ Again this applies:
 http://www.osnews.com/story/19266/WTFs_m
 
 
+The review-change-push-review cycle
+-----------------------------------
+
+After you have made a pull request, you should get a review via the github interface. After the review you will frequently have to do some changes of your PR, add a missing changelog entry, and so on.
+When the changes are done, rebase your branch again, squash commits, all as described above.
+Finally you should force push your feature branch to gitlab. Only force push on your own feature branches! Never on branches shared with other people.
```
Comment (thet) [inline docs/git.rst]: GitHub instead of gitlab.
Also please use the correct ``GitHub`` camel case wording instead ``github`` everywhere.

**Review by thet — CHANGES_REQUESTED** [review]
Just a minor comment regarding naming. Otherwise LGTM.

**Comment by sunew** [comment]
@thet done:)

File: docs/git.rst [hunk 2]
```diff
@@ -560,6 +560,15 @@ Again this applies:
 http://www.osnews.com/story/19266/WTFs_m
 
 
+The review-change-push-review cycle
+-----------------------------------
+
+After you have made a pull request, you should ask for a review via the GitHub interface. After the review you will frequently have to do some changes of your PR, perhaps add a missing changelog entry, and so on.
```
Comment (jensens) [inline docs/git.rst]: Please follow the doc-guidelines, which here means here to use semantic linefeeds. See https://docs.plone.org/about/contributing/documentation_styleguide.html#line-length

**Review by jensens — CHANGES_REQUESTED** [review]

File: docs/git.rst (hunk 2 above)
Comment (sunew) [reply docs/git.rst]: @jensens done:)

**Comment by sunew** [comment]
@jensens I modified according to your review - ready to merge?

**Comment by jensens** [comment]
Yes. Thanks!

**Review by jensens — APPROVED** [review]

**Pull request merged**